outbox-spring/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
```

Comment:
Should configs follow DRY principle for Lombok, JUnit versions?

Just a remark - may not be appropriate to decide at this time